content/acra/configuring-maintaining/general-configuration/acra-keys/destroy.md

```diff
@@ -367,20 +367,17 @@ Make sure you have set `ACRA_MASTER_KEY` env variable for keystore `v1`.
 {{< /hint >}}
 
 ```
-$ acra-keys generate --client_id=user1 --keystore=v1 --acraconnector_transport_key --acraserver_transport_key
+$ acra-keys generate --client_id=user1 --keystore=v1 --search_hmac_symmetric_key
```

Comment:
some rows above we have:
> For example, lets generate several transport keys using ...

But the example about generation keys for searchable encryption. Should be rephrased.